baremetal/metal3data_manager_test.go

```diff
@@ -1241,7 +1290,7 @@ var _ = Describe("Metal3Data manager", func() {
 			poolRef:         corev1.TypedLocalObjectReference{Name: testPoolName},
 			expectedAddress: addressFromPool{},
 			ipClaim: &ipamv1.IPClaim{
-				ObjectMeta: testObjectMeta("abc-abc", namespaceName, ""),
+				ObjectMeta: testObjectMeta(metal3DataName+"-"+testPoolName, namespaceName, ""),
```

Comment:
The name was wrong here so the test went to "IPClaim not found" instead of finding the IPClaim and then not fining the IPAddress.